libs/core/src/lib/utils/drag-and-drop/dnd-item/dnd-item.directive.ts

```diff
@@ -0,0 +1,255 @@
+import { AfterContentInit, Directive, ElementRef, EventEmitter, Input, OnDestroy, Output } from '@angular/core';
+import { DragDrop, DragRef } from '@angular/cdk/drag-drop';
+import { ElementChord, LinkPosition } from '../dnd-list/dnd-list.directive';
+import { Subscription } from 'rxjs';
+
+export interface ElementPosition {
+    x: number;
+    y: number;
+}
+
+@Directive({
+    // tslint:disable-next-line:directive-selector
+    selector: '[fd-dnd-item]',
+    host: {
+        class: 'fd-dnd-item'
+    },
+    providers: [
+        DragDrop
+    ]
+})
+export class DndItemDirective implements AfterContentInit, OnDestroy {
+    /** Event thrown when the element is moved by 1px */
+    @Output()
+    readonly moved = new EventEmitter<ElementPosition>();
+
+    /** Event thrown when the element is released */
+    @Output()
+    readonly released = new EventEmitter<void>();
+
+    /** Event thrown when the element is started to be dragged */
+    @Output()
+    readonly started = new EventEmitter<void>();
+
+    /**
+     *  Defines if the item is prevented from being moved by other elements.
+     * So nothing can be placed just before and just after it
+     */
+    @Input()
+    stickInPlace = false;
+
+    /** Defines if element is draggable */
+    @Input()
+    set draggable(draggable: boolean) {
+        this._draggable = draggable;
+        this.changeCDKDragState();
+    }
+
+    /** Class added to element, when it's dragged. */
+    @Input()
+    classWhenElementDragged = 'fd-dnd-on-drag';
+
+    /** Defines if every element in list is draggable */
+    listDraggable = true;
+
+    /** @hidden
+     * Drag reference, object created from DND CDK Service
+     */
+    private _dragRef: DragRef;
+
+    /** @hidden */
+    private _draggable = true;
+
+    /** @hidden */
+    private _subscriptions = new Subscription();
+
+    /** @hidden */
+    private _placeholderElement: HTMLElement;
```

Comment:
I think we should name anything with placeholder differently, it is confusing

Reply:
The naming is same as on angular CDK. I think placeholderElement is meaningful. I can add some additional comment there, that it's added as a ghost element, that indicates position of dragged element.